src/lib/seam/connect/api-error.ts

```diff
@@ -0,0 +1,30 @@
+// UPSTREAM: Should be provided by @seamapi/types.
+interface ApiError {
+  type: string
+  message: string
+  data?: unknown
+}
+
+export class SeamHttpApiError extends Error {
+  code: string
```

Comment:
Why `code` and not `type`? This is actually the standard property for Node.js errors: https://nodejs.org/api/errors.html#errorcode

Type is not quite correct here, since the type is really just the class name: SeamHttpApiError